src/main/java/org/broadinstitute/hellbender/tools/exome/sexgenotyper/TargetCoverageSexGenotyper.java

```diff
@@ -144,47 +153,68 @@
     protected double baselineMappingErrorProbability = 1e-4;
 
     @Argument(
-            doc = "Input target list. A tab-separated list of targets used for sex genotyping. " +
-                    " It must be a subset or can be all of the targets included in the read counts table.",
+            doc = "Input targets file. A tab-separated (.tsv) list of targets (not a BED file) with at least" +
```

Comment:
I see, like the M2ArgumentCollection. Sounds good to me. I didn't touch much of these descriptions because this is the beta release and we will have a more concerted effort towards standardization after the workshop tutorial work. So let me know what I can do to help.